ITP/symmetry-breaking.tex

```diff
@@ -1,32 +1,50 @@
+    \item \textbf{($x$-order)} The points are sorted with respect to their $x$-coordinates, i.e., $x(p_i) < x(p_j)$ for all $1 \leq i < j \leq n$.
     \item \textbf{(CCW-order)} All orientations $\sigma(p_1, p_i, p_j)$, with $1 < i < j \leq n$, are counterclockwise.
-    \item \textbf{(Lex order)} The list of orientations \( \left(\sigma\left(p_{\lceil \frac{n}{2} \rceil -1}, p_{\lceil \frac{n}{2} \rceil},p_{\lceil \frac{n}{2} \rceil+1}\right), \ldots, \sigma\left(p_2, p_3, p_4\right) \right)\) is not lexicographically smaller than the list \(\left(\sigma\left(p_{\lfloor \frac{n}{2} \rfloor  + 1}, p_{\lfloor \frac{n}{2} \rfloor+2},p_{\lfloor \frac{n}{2} \rfloor+3}\right), \ldots, \sigma\left(p_{n-2}, p_{n-1}, p_{n}\right) \right).\)
+    \item \textbf{(Lex-order)} The list of orientations \( \left(\sigma\left(p_{\lceil \frac{n}{2} \rceil -1}, p_{\lceil \frac{n}{2} \rceil},p_{\lceil \frac{n}{2} \rceil+1}\right), \ldots, \sigma\left(p_2, p_3, p_4\right) \right)\) is not lexicographically smaller than the list \(\left(\sigma\left(p_{\lfloor \frac{n}{2} \rfloor  + 1}, p_{\lfloor \frac{n}{2} \rfloor+2},p_{\lfloor \frac{n}{2} \rfloor+3}\right), \ldots, \sigma\left(p_{n-2}, p_{n-1}, p_{n}\right) \right).\)
     % Given the general position condition, all orientations are in $\{-1, 1\}$, and thus the lexicographic condition is equivalent to stating that there is an index $i$ such that $\forall j < i$, $\textsf{Left}[j] = \textsf{Right}[j]$, and $\textsf{Left}[i] = -1$ but $\textsf{Right}[i] = 1$.
 \end{itemize}
 \end{definition}
 
-This symmetry breaking assumption not only reduces the search space of the SAT solver, but it is required for the correctness of the encoding,
-as clauses~\labelcref{eq:insideClauses1,eq:insideClauses2,eq:holeDefClauses1,eq:signotopeClauses11,eq:signotopeClauses12} rely on points being sorted from left to right.
-Before discussing the proof of correctness of symmetry breaking,
-let us first focus on the last condition, expressed explicitly in clause~\labelcref{eq:revLexClauses} of the encoding.
-The main idea behind this condition is that \emph{reflecting} a pointset, i.e., applying the map $(x, y) \mapsto (-x, y)$,
-preserves the presence of $k$-holes, or convex $k$-gons.
-This is the reason for incorporating the \emph{parity} flag in the definition of $\sigma$-equivalence.
-\input{fig-sigma-equiv.tex}
-As illustrated in~\Cref{fig:sigma-equiv}, there are pointsets that are only $\sigma$-equivalent to their reflections
-with \lstinline|parity := true|.
-We are now ready to state the main symmetry breaking theorem and sketch its proof.
+The three ordering properties each break a different symmetry.
+First, the $x$-order property breaks symmetry due to how we label the points by ensuring that the points are labeled from left to right.
+Plus,
+the $x$-order does more than break symmetry:
```

Comment:
this seems potentially confusing to the reader -- maybe say "this assumption also substantially simplifies the encoding of clauses~\labelcref{...}"

Reply:
Incorporated.